R/graph.loglik.R

```diff
@@ -6,7 +6,7 @@
 #' @param log_x should the x-axis be on a logarithmic scale (`TRUE`) or linear scale (`FALSE`, default)?
 #' @param previous_plot if not NULL, the current data is added to the existing graph
 #' @param curve_label if not NULL, add a label for the curve
-#' @inheritParams llik
+#' @inheritParams log_likelihood
 #' @inheritDotParams llik -lambda
```

Comment:
update `llik` -> `log_likelihood` on line 10?

Reply:
This change has been made.

Reply:
@chrisorwa thanks! looks like this PR is complete. Please increase the version number (using `usethis::use_version()`; minor version increment) and summarize the changes in NEWS.md, and I'll approve.